package.json

```diff
@@ -19,6 +18,7 @@
     "start": "npm run serve:dev src",
     "lint": "eslint --cache src test",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
+    "build:watch": "npm run -s build -- -w",
```

Comment:
Better to replace it with the current `start` script, I think we were going this route for `src/tasks/package.js` already and it's just missing here. Please check that out :)